sql/backfill.go

```diff
@@ -152,6 +155,51 @@ func (p *planner) backfillBatch(b *client.Batch, tableDesc *TableDescriptor) *ro
 		b.DelRange(indexStartKey, indexEndKey, false)
 	}
 
+	// Add new columns.
+	if len(newColumnDescs) > 0 {
+		// Create default expressions
+		defaultExprs := make(map[ColumnDescriptor]parser.Expr)
+		for _, columnDesc := range newColumnDescs {
+			tableDesc.AddColumn(columnDesc)
+			d, err := p.makeDefaultExprs([]ColumnDescriptor{columnDesc})
+			if err != nil {
+				return roachpb.NewError(err)
+			}
+			if columnDesc.DefaultExpr == nil && !columnDesc.Nullable {
+				return &roachpb.Error{
+					Message: fmt.Sprintf("column %s contains null values", columnDesc.Name),
+				}
+			}
+			if len(d) > 0 {
+				defaultExprs[columnDesc] = d[0]
+			}
+		}
+
+		for _, result := range batch.Results {
+			var sentinelKey roachpb.Key
```

Comment:
This part is mostly copy-pasta from above. Is it possible to combine the two? I think that would be preferable (also because once we batch the backfill, it'll be better to batch on operations touching the same keys simultaneously).